### DISCUSSION TITLE
官方將有機會發布官方版的繁體中文語言包

### DISCUSSION THREAD

**Discussion by bluelovers** [bluelovers/idea-l10n-zht#11] [Announcements]
請參考
- https://youtrack.jetbrains.com/issue/MP-4217
- https://github.com/bluelovers/idea-l10n-zht/issues/7

https://plugins.jetbrains.com/plugin/18365-chinese-traditional-language-pack-----

**Comment by bluelovers** [comment]
![image](https://user-images.githubusercontent.com/167966/153692719-5470f91b-2434-4dde-8f2d-edaeba398ce1.png)

**Comment by bluelovers** [comment]
@ruyut 官方今天回應是 因為安全性問題 所以不允許發布
但會藉由這個機會發布一個官方版的繁體中文語言包

並且能夠接受去修貢獻翻譯

> There’s a demand for a Traditional Chinese language pack, and we hope we can work together to help our users who need this language pack.
> We’ll appreciate it if you could help us with translating our files into Traditional Chinese, and JetBrains will keep maintaining and updating the plugin making sure it is compatible with our latest available versions.

**Comment by bluelovers** [comment]
結果官方版本的做法是透過 crowdin 來翻譯

但實際上打包程式似乎又是透過其他方式
導致經常發生更新後會把原本的翻譯不見的狀況

或者 檔名大小寫錯誤這種情形

**Reply by SiongSng** [reply]
Crowdin 應該不是大問題，我之前用過還不錯用，只是要看他的打包程式怎麼寫的吧

**Reply by bluelovers** [reply]
> Crowdin 應該不是大問題，我之前用過還不錯用，只是要看他的打包程式怎麼寫的吧


目前問題卡在 我上傳語言包給官方之後 完全沒變化 永遠都是翻譯進度 0% 點進去看 也沒有更新翻譯 都還是原文

如果要我一個一個手動編輯的話 我受不了XD

**Reply by bluelovers** [reply]
> Crowdin 應該不是大問題，我之前用過還不錯用，只是要看他的打包程式怎麼寫的吧

一開始想用 crowdin api 來更新翻譯 但發現好像沒有權限 可以這樣做

**Reply by SiongSng** [reply]
你是從 Crowdin 的 UI 上傳檔案嗎?
我自己有做過使用 Crowdin API 來自動上傳的腳本

**Reply by bluelovers** [reply]
> 你是從 Crowdin 的 UI 上傳檔案嗎? 我自己有做過使用 Crowdin API 來自動上傳的腳本

恩 透過 UI 傳上去的好像都失敗

https://crowdin.com/project/community-intellij-inspections-and-inten/zh-TW

https://crowdin.com/project/community-intellij/zh-TW

**Reply by SiongSng** [reply]
Something went wrong
Project does not exist or access to this project is limited.

我點進去之後顯示這樣，是不是沒對外開放?

**Reply by bluelovers** [reply]
> Something went wrong Project does not exist or access to this project is limited.
> 
> 我點進去之後顯示這樣，是不是沒對外開放?

恩 好像是邀請制的

**Comment by bluelovers** [comment]
相關的官方回報帖

didn't have download/export button in `community-intellij`
https://youtrack.jetbrains.com/issue/IDEA-289876

upload Chinese Traditional translation to `community-intellij-inspections-and-inten` didn't see change
https://youtrack.jetbrains.com/issue/IDEA-290029
